test/test_ops.py

```diff
@@ -451,9 +451,6 @@ def out_fn(output):
 
     @_alias_ops((op for op in op_db if op.aliases))
     def test_jit_alias_remapping(self, device, dtype, op):
-        # Local import needed for test scripting
-        from torch import tensor
-
```

Comment:
This was needed before but tests surprisingly pass now. Must be some changes in upstream! 🎉